api/api/admin/__init__.py

```diff
@@ -47,32 +57,214 @@ class MediaReportAdmin(admin.ModelAdmin):
         if request.path.endswith("/change/"):
             return ["media_obj"]
 
+    def get_fieldsets(self, request, obj=None):
+        if obj is None:
+            return [
+                (
+                    "Report details",
+                    {"fields": ["status", "decision", "reason", "description"]},
+                ),
+                ("Media details", {"fields": ["media_obj"]}),
+            ]
+        return [
+            (
+                "Report details",
+                {
+                    "fields": [
+                        "created_at",
+                        "status",
+                        "decision",
+                        "reason",
+                        "description",
+                        "has_sensitive_text",
+                    ],
+                },
+            ),
+            (
+                "Media details",
+                {
+                    "fields": [
+                        "media_display",
+                        "title",
+                        "tags",
+                        "media_description",
+                        "from_field",
+                        "landing_urls",
+                        "media_obj_id",
+                    ]
+                },
+            ),
+            ("Other reports", {"fields": ["other_reports"]}),
+        ]
+
     def get_readonly_fields(self, request, obj=None):
         if obj is None:
             return []
         readonly_fields = [
+            "created_at",
             "reason",
             "description",
+            "has_sensitive_text",
+            "media_display",
+            "title",
+            "tags",
+            "media_description",
+            "from_field",
+            "landing_urls",
             "media_obj_id",
-            "created_at",
+            "other_reports",
         ]
         # ``status`` cannot be changed on a finalised report.
         if obj.status != PENDING:
             readonly_fields.append("status")
         return readonly_fields
 
 
-admin.site.register(AudioReport, MediaReportAdmin)
-admin.site.register(ImageReport, MediaReportAdmin)
+@admin.register(ImageReport)
+class ImageReportAdmin(MediaReportAdmin):
+    change_form_template = "admin/api/imagereport/change_form.html"
+    media_type = "image"
+
+    @admin.display(description="Image")
+    def media_display(self, obj):
+        """
+        Display a blurred image with a clickable overlay.
+        Use the image thumbnail if available, else replace with the direct image url.
+        """
+        if obj.media_obj.url:
+            thumb_url = f"https://api.openverse.engineering{obj.media_url()}thumb/"
+            return mark_safe(
+                f'<div class="container"><img src="{thumb_url}" alt="Media Image" class="blur-image" height="300" '
+                f'onclick="toggleBlur(this)" onerror="replace(this, \'{obj.media_obj.url}\')" style="cursor: pointer;" />'
```

Comment:
What is replace doing here in the `onerror`? The implementation of the function doesn't clear anything up to me. Why would the `img` element's `src` ever need to change to point to a non-image URL?

Reply:
By default, we use the thumbnail url. However, especially locally, this URL might not contain an image: some of the sample data identifiers do not exist in the production database. So, for local testing, we try loading the thumbnail url as the image src. If that cannot be loaded due to image not being available at that URL, we try loading the image's direct URL, which most likely exists. 
This behavior mirrors the behavior of the frontend single result image, where we try to load the direct image URL, and if that is not available - we load the thumbnail.

We should update the sample data to only contain data that exists in production, even if the links are dead - at least we should have the items for all identifiers in the sample data - but that's a different issue.

Reply:
Ohhhh I see, I totally misread `obj.media_obj.url`, and interpreted it as the API route for the image rather than the upstream image URL.

Okay, so I understand this now, thanks for clarifying. The other question is then... do we need to do anything to make sure this works as expected for both images and audio (audio thumbnail doesn't proxy `.url`, I think? :thinking:)